src/ansiblelint/_prerun.py

```diff
@@ -226,9 +226,14 @@ def _prepare_ansible_paths() -> None:
 
 def _make_module_stub(module_name: str) -> None:
     # a.b.c is treated a collection
-    if re.match(r"\w+\.\w+\.\w+", module_name):
-        namespace, collection, module_file = module_name.split(".")
+    if re.match(r"\w+\.\w+\.[\.\w]+$", module_name):
```

Comment:
Shouldn't this be
```suggestion
    if re.match(r"\w+\.\w+(\.\w+)+$", module_name):
```
if you want to match proper names?